package.json

```diff
@@ -36,9 +36,6 @@
   },
   "author": "The Betaflight open source project.",
   "license": "GPL-3.0",
-  "engines": {
```

Comment:
Deleting this does not make any sense - it will just bring back 'bug reports' from users trying to use an unsupported version of node.